src/train.py

```diff
@@ -83,16 +88,33 @@ def validate(net, epoch_num):
     print('\n\n\n')
 
 
+def charVector(batch,char=None):
+    # todo fix tensor
+    if not char:
+        char = np.random.randint(0, 70, batch)
+    
+    onehotchar = np.zeros(70)
+    onehotchar[char] = 1
```

Comment:
Be careful: we've been bitten in the past by incorrect datatypes. This looks like its `np.int64`. Do you intend for it to be `np.float64` instead?

Reply:
i think we want it to be an int, bc int corresponds with the char index